src/__tests__/anchored-position.test.ts

```diff
@@ -366,4 +366,19 @@ describe('getAnchoredPosition', () => {
     expect(top).toEqual(54) // anchorRect.top + anchorRect.height + (settings.anchorOffset ?? 4) - parentRect.top
     expect(left).toEqual(380) // anchorRect.left + anchorRect.width - parentRect.left - floatingRect.width
   })
+
+  // This test runs for values derived from a real use case https://github.com/github/accessibility-audits/issues/4515 as run on a local storybook.
+  it('should overflow to bottom if the element is too tall to fit on the screen when zoomed', () => {
+    const parentRect = makeDOMRect(0, 0, 400, 400)
+    const anchorRect = makeDOMRect(16, 16, 32, 32) // left aligned button
+    const floatingRect = makeDOMRect(0, 0, 256, 428)
+    const {float, anchor} = createVirtualDOM(parentRect, anchorRect, floatingRect)
+    const settings: Partial<PositionSettings> = {side: 'outside-bottom', align: 'start'}
+    const {top, left, anchorSide, anchorAlign} = getAnchoredPosition(float, anchor, settings)
+    // Not really sure what to expect here.
+    expect(anchorSide).toEqual('outside-right')
```

Comment:
Generally speaking the implementation tries to fit the element in the available screen real estate, so it may move to the right/left/top if there's enough space for it to move there, otherwise if it cannot fit anywhere it should move to the outside-bottom.

I think as you fix the `top` this will probably end up being `outside-bottom`. Which is fine.

Reply:
Yes, sorry for the misleading comment. I was only interested for the value of `top` to be `>0` for this test case